zcash_client_sqlite/CHANGELOG.md

```diff
@@ -10,11 +10,24 @@ and this library adheres to Rust's notion of
 ### Added
 - A new `orchard` feature flag has been added to make it possible to
   build client code without `orchard` dependendencies.
+- `zcash_client_sqlite::AccountId` 
 - `impl From<zcash_keys::keys::AddressGenerationError> for SqliteClientError`
 
 ### Changed
+- Many places that `AccountId` appeared in the API changed from
+  using `zcash_primitives::zip32::AccountId` to using an opaque `zcash_client_sqlite::AccountId`
+  type. This impacts the following APIs:
+  - `zcash_client_sqlite::error::SqliteClientError::AccountUnknown`
+  - Changes to the implementation of the `WalletWrite` trait:
+    - `create_account` function returns a unique identifier for the new account (as before),
+      except that this ID no longer happens to match the ZIP-32 account index.
+      To get the ZIP-32 account index, use the new `WalletRead::get_account` function.
+  - Two columns in the `transactions` view were renamed:
```

Comment:
Were they just renamed, or were their contents altered as well to no longer be ZIP 32 account indices? If the latter, explain this.

Reply:
Clarified.